src/app/payment/hooks/useUserPayment.ts

```diff
@@ -55,13 +56,15 @@ export const useUserPayment = () => {
     priceId,
     currency,
     token,
+    userAddress,
```

Comment:
Are you referring to the user's IP address? If so, I think it would be better to change it to `userIpAddress`